docroot/themes/humsci/humsci_basic/package.json

```diff
@@ -28,13 +28,13 @@
   "license": "MIT",
   "dependencies": {
     "bourbon": "^5.1.0",
-    "decanter": "^6.2.12",
+    "decanter": "^6.2.19",
```

Comment:
can you update to decanter 6.3+?

Reply:
We are planning to upgrade to 6.3 in the near future, however we need some extra time for that because that version contains breaking changes that might affect the current theme and we want to do extra testing to confirm everything works as expected. For now, we decided to update  to the latest 6.2 version, to use the updated version of Source Sans and Source Serif fonts.